packages/flutter_tools/bin/macos_assemble.sh

```diff
@@ -58,7 +58,7 @@ BuildApp() {
   # arguments.
   local flutter_args=("${FLUTTER_ROOT}/bin/flutter")
   if [[ -n "$VERBOSE_SCRIPT_LOGGING" ]]; then
-    flutter_args+=('--verbose')
+  flutter_args+=('--verbose')
```

Comment:
was this de-dent intentional?

Reply:
Mistake, you beat me reverting it by a second.

Reply:
lol, yeah, I saw that

Reply:
It's Friday evening and we're on fire. 🔥